### ISSUE TITLE
Featured News links should read link text instead of alt text (Level A failure)

### ISSUE THREAD

**Issue by mmafe** [ccswbs/ugnext#358]
Featured news links should read link text (including category) instead of alt text.

<img width="1319" height="758" alt="Image" src="https://github.com/user-attachments/assets/a296643a-acbe-4bd5-9d00-2046acc3d470" />

**Comment by mmafe** [comment]
Fixed
